userspace/engine/stats_manager.h

```diff
@@ -63,7 +63,6 @@ class stats_manager
 		const indexed_vector<falco_rule>& rules,
 		std::string& out) const;
 
-private:
 	std::atomic<uint64_t> m_total;
 	std::vector<std::unique_ptr<std::atomic<uint64_t>>> m_by_priority;
 	std::vector<std::unique_ptr<std::atomic<uint64_t>>> m_by_rule_id;
```

Comment:
Outside of this PR scope but why unique pointers to atomic rather than just atomics ?

Reply:
cc @jasondellaluce ?